docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
       - OTEL_EXPORTER_OTLP_ENDPOINT=http://jaeger:4317
       - OTEL_RESOURCE_ATTRIBUTES=service.name=datajunction.server
       - DOTENV_FILE=${DOTENV_FILE:-/code/.env}
+      - OAUTHLIB_INSECURE_TRANSPORT=1
```

Comment:
Where is this variable used?

Reply:
This is picked up by google's oauth python library. It's just a flag that relaxes the https requirement so that we can try it out on http://localhost